kloppy/domain/models/position.py

```diff
@@ -92,3 +92,10 @@ def __str__(self):
     @classmethod
     def unknown(cls) -> "PositionType":
         return cls.Unknown
+
+
+class RefereeType:
+    VideoReferee = "Video Referee"
+    Referee = "Referee"
```

Comment:
They are all a referee, hence using `RefereeType="Referee"` seems ambiguous.

```suggestion
    Referee = "Main Referee"
```